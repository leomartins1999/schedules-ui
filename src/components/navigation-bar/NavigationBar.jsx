import Navbar from 'react-bootstrap/Navbar';
import Nav from 'react-bootstrap/Nav';
import Container from 'react-bootstrap/Container';

function NavigationBar() {
    return (
        <Navbar bg="primary" variant="dark">
            <Container>
                <Navbar.Brand>Schedule Comparator</Navbar.Brand>
                <Nav className="me-auto">
                    <Nav.Link href="#home">Home</Nav.Link>
                    <Nav.Link href="#schedules">Schedules</Nav.Link>
                    <Nav.Link href="#compare">Compare</Nav.Link>
                </Nav>
            </Container>
        </Navbar>
    );
}

export default NavigationBar;
