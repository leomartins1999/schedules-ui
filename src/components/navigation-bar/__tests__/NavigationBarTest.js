import renderer from 'react-test-renderer';
import NavigationBar from "../NavigationBar"

it('renders the NavBar', () => {
    const tree = renderer
        .create(<NavigationBar />)
        .toJSON();

    expect(tree).toMatchSnapshot();
})
