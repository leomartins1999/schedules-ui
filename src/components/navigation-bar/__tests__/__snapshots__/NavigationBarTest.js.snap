// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the NavBar 1`] = `
<nav
  className="navbar navbar-expand navbar-dark bg-primary"
>
  <div
    className="container"
  >
    <span
      className="navbar-brand"
    >
      Schedule Comparator
    </span>
    <div
      className="me-auto navbar-nav"
      onKeyDown={[Function]}
    >
      <a
        className="nav-link"
        data-rr-ui-event-key="#home"
        disabled={false}
        href="#home"
        onClick={[Function]}
      >
        Home
      </a>
      <a
        className="nav-link"
        data-rr-ui-event-key="#schedules"
        disabled={false}
        href="#schedules"
        onClick={[Function]}
      >
        Schedules
      </a>
      <a
        className="nav-link"
        data-rr-ui-event-key="#compare"
        disabled={false}
        href="#compare"
        onClick={[Function]}
      >
        Compare
      </a>
    </div>
  </div>
</nav>
`;
